### PR TITLE
PIA-1915: CI Fix 

### PR DESCRIPTION
**Pull request by kp-geneva-parayno**
**Summary**
Removed parallel running
Removed before and after suite

### CODE REVIEW

**Comment by kp-geneva-parayno** [comment]
testing tvos ci - do not review

File: PIA-VPN_E2E_Tests.xctestplan [hunk 1]
```diff
@@ -12,15 +12,15 @@
...
       },
       {
         "key" : "PIA_TEST_PASSWORD",
-        "value" : "xxx"
+        "value" : "GxLVXiSV5s"
       },
       {
         "key" : "PIA_TEST_DEDICATEDIP",
-        "value" : "xxx"
+        "value" : "DIP2B3lNYGmcQHvOyrByYZ7TWKztfYLQ"
```
Comment (kp-laura-sempere) [inline PIA-VPN_E2E_Tests.xctestplan:23]: @kp-geneva-parayno , we probably don't want to commit these. Those should be passed in the env variables for CI

**Review by kp-laura-sempere — COMMENTED** [review]
LGMT, I just added one comment on the xctestplan
Once that's addressed, this can be merged
Thank you!

File: PIA-VPN_E2E_Tests.xctestplan (hunk 1 above)
Comment (kp-geneva-parayno) [reply PIA-VPN_E2E_Tests.xctestplan]: updated :)

**Comment by kp-geneva-parayno** [comment]
thanks @kp-laura-sempere  - updated. 
not sure why the tvos unit tests are failing?

**Review by kp-said-rehouni — APPROVED** [review]
LGTM! Good job @kp-geneva-parayno!

**Pull request merged**